src/limb.rs

```diff
@@ -60,7 +60,7 @@ fn limbs_less_than_limbs(a: &[Limb], b: &[Limb]) -> Result<LimbMask, LenMismatch
     // optimizer.
     // XXX: Questionable whether `LenMismatchError` is appropriate.
     let len = NonZeroUsize::new(b.len()).ok_or_else(|| LenMismatchError::new(a.len()))?;
-    if a.len() != len.into() {
+    if a.len() != <NonZero<usize> as Into<usize>>::into(len) {
```

Comment:
Thank you for the PR.

Instead, we should use `len.get()` instead of `len.into()`. Your change won't work with old versions of Rust we support because they don't have `NonZero<T>`.